client/src/pages/Login.jsx

```diff
@@ -5,58 +5,39 @@ import { LOGIN } from "../utils/mutations";
 import Auth from "../utils/auth";
 import { Button, Typography, Input, Card, FormHelperText } from "@mui/joy";
 import { useForm } from "react-hook-form";
-import { useSchemas } from "../hooks/useSchemas";
-import { yupResolver } from "@hookform/resolvers/yup";
-
-const styles = {
-  card: {
-    width: 400,
-    mx: "auto",
-    p: 2,
-    mt: 4,
-    my: 4,
-    borderRadius: "sm",
-    display: "flex",
-    flexDirection: "column",
-    gap: 2,
-    boxShadow: "lg",
-    backgroundColor: "lightblue",
-  },
-};
+import { styles } from "../styles/cardstyles";
 
```

Comment:
I figured out how to do the comments in the code :)

I've moved the individual styles to a separate file. For the time being, there are going to be individual style files for each page but eventually they will be consolidated into a theme and accessed through context.